.github/workflows/tests/install_script_tests.bats

```diff
@@ -1,99 +1,138 @@
 #!/usr/bin/env bats
 
+setup() {
+    echo '
+#!/bin/bash
+
+# Define the version string
+version="2024.9"
+
+# Check if the --version argument is passed
+if [[ "$1" == "--version" ]]; then
+  echo "rpm-ostree:"
+  echo " Version: '\''$version'\''"
+else
+  # Default behavior for unknown arguments (if you want to handle them)
+  echo "Invalid option. Usage: rpm-ostree --version"
+fi
+    ' > rpm-ostree
+    chmod +x rpm-ostree
+    sudo cp -f rpm-ostree /usr/bin/rpm-ostree
+
+}
+
 @test "Script exits with error if rpm-ostree is not installed" {
-  sudo bash -c 'rm -f /usr/bin/rpm-ostree'
+  sudo bash -c 'mv /usr/bin/rpm-ostree /usr/bin/rpm-ostree.backup'
   run bash "$INSTALL_SCRIPT"
   [ "$status" -eq 1 ]
   [[ "$output" == *"This script only runs on Fedora Atomic"* ]]
+  sudo bash -c 'mv /usr/bin/rpm-ostree.backup /usr/bin/rpm-ostree'
+}
+
+@test "Script exits with error if rpm-ostree is very old" {
+  sudo sed -i 's/2024\.9/2023\.5/' /usr/bin/rpm-ostree
+  run bash "$INSTALL_SCRIPT"
+  [ "$status" -eq 1 ]
+  [[ "$output" == *"rpm-ostree is too old, please upgrade before running this script."* ]]
+  sudo sed -i 's/2023\.5/2024\.9/' /usr/bin/rpm-ostree
+}
+
+@test "Script exits with error if rpm-ostree is slightly old" {
+  sudo sed -i 's/2024\.9/2024\.8/' /usr/bin/rpm-ostree
+  run bash "$INSTALL_SCRIPT"
+  [ "$status" -eq 1 ]
+  [[ "$output" == *"rpm-ostree is too old, please upgrade before running this script."* ]]
+  sudo sed -i 's/2023\.8/2024\.9/' /usr/bin/rpm-ostree
```

Comment:
I believe this should be `2024\.8`, not 2023

Reply:
good catch